src/Mutex/Activities.cs

```diff
@@ -0,0 +1,38 @@
+using Temporalio.Activities;
+
+namespace TemporalioSamples.Mutex;
```

Comment:
Ideally the namespace is at the top, but don't think it matters that much.

Reply:
Currently, `.editorconfig` does not contain `csharp_using_directive_placement`. Should I add `csharp_using_directive_placement = inside_namespace`?

Reply:
Cleaned up 'usings'